spec/inputs/gelf_spec.rb

```diff
@@ -1,8 +1,22 @@
 require "logstash/devutils/rspec/spec_helper"
 require "logstash/inputs/gelf"
+require_relative "../support/helpers"
 require "gelf"
 
-describe "inputs/gelf" do
+Thread.abort_on_exception = true
+describe LogStash::Inputs::Gelf do
+  context "when interrupting the plugin" do
+    let(:port) { "12333" }
```

Comment:
unless we randomize this we'll probably have issues when jenkins tests the multiple jvms in parallel